### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dharamgollapudi**
Proposed changes include grammatical corrections and converting TODO list into Task list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ JVips, a libvips Java wrapper
 
 ![Logo](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/emojidex/112/high-voltage-sign_26a1.png)
 
-JVips is a Java wrapper around [libvips](https://github.com/libvips/libvips) using JNI.
+JVips is a Java wrapper for [libvips](https://github.com/libvips/libvips) using JNI.
```
Comment (dharamgollapudi) [inline README.md:6]: wrapper already implies around

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ JVips, a libvips Java wrapper
 
 ![Logo](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/emojidex/112/high-voltage-sign_26a1.png)
 
-JVips is a Java wrapper around [libvips](https://github.com/libvips/libvips) using JNI.
```
Comment (dharamgollapudi) [inline README.md:6]: wrapper already implies around

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ $ docker run --rm -v $(pwd):/app -w /app jvips-builder-linux
 
 ### 🏁 Windows
 
-Windows builds are cross-compiled from Linux using MingW64. WSL and Docker are both supported.
```
Comment (dharamgollapudi) [inline README.md:118]: Ordered them matching the instructions below

File: README.md [hunk 4]
```diff
@@ -208,10 +208,10 @@ $ docker run --rm -v $(pwd):/app -w /app -u root -it builder bash
 
 ## TODO
 
-- Add the missing operations
```
Comment (dharamgollapudi) [inline README.md:211]: Adding `[ ]` to signify the pending tasks when it is rendered.

**Comment by dbouron** [comment]
Hello,

Thank you for your changes, LGTM!

**Pull request merged**